src/bin/casr-cli.rs

```diff
@@ -629,3 +660,217 @@ fn change_text_view(layout1: &mut LinearLayout, act: Action) -> Option<EventResu
         Action::Mouse(_) => None,
     }
 }
+
+/// Print common statistic all over reports in directory
+///
+/// # Arguments
+///
+/// * 'dir' - directory with reports
+///
+fn print_summary(dir: PathBuf) -> Result<()> {
```

Comment:
We don't return any errors. Maybe remove Result?

Reply:
done